ACPs/226-dynamic-minimum-block-times/README.md

```diff
@@ -82,7 +88,7 @@ Parameters at activation on the C-Chain are:
 | Parameter | Description | C-Chain Configuration|
 | - | - | - |
 | $M$ | minimum `minimumBlockDelay` value | 100 milliseconds |
```

Comment:
I think we should also make this 1ms to align with potential subnet-evm implementations.